nomenclature/definitions/variable/product_types.yaml

```diff
@@ -0,0 +1,206 @@
+# This list of products will replace any occurrence of `<product>`
+# in a variable-definition yaml file
+
+
+#   i.e., The following standard structure in economic variables:
+#   Consumption|Households|< product >|Imported:
+#   Consumption|Government|< product >|Domestic:
+
+<product>:
+  pPLNT:
+    description: Crops and vegatable in agri sector
+
+  pANIM:
```

Comment:
Dear @hettieboonman, I'm confused about the prefixes `p` and `n` in each product. 
Could you explain us a little about ti, please?
I'll try to put your explanation as a clarification in this `product_types.yaml` file.